assemble/bin/accumulo-cluster

```diff
@@ -646,12 +651,171 @@ function control_services() {
   if [[ $ARG_LOCAL == 0 && $ARG_ALL == 1 && ($operation == "stop" || $operation == "kill") ]]; then
     if ! isDebug; then
       echo "Cleaning all server entries in ZooKeeper"
-      "$accumulo_cmd" org.apache.accumulo.server.util.ZooZap -manager -tservers -compaction-coordinators -compactors -sservers
+      "$accumulo_cmd" org.apache.accumulo.server.util.ZooZap -manager -tservers -compaction-coordinators -compactors -sservers --gc --monitor
     fi
   fi
 
 }
 
+function prune_group() {
+  local service_type=$1
+  local group=$2
+  local expectedCount=$3
+  shift 3
+  local hosts=("$@")
```

Comment:
This doesn't need to shift, and this should be unquoted to convert the scalar into an array. shellcheck will probably complain here, but you can add an exception, or you can do something else to read the scalar into an array using some other means

```suggestion
  local hosts=($4)
```